common/source/docs/common-blheli32-passthru.rst

```diff
@@ -3,15 +3,17 @@
 BLHeli_32 Pass-Through Support
 ==============================
 
+BLHeli capable ESCs have the ability to have their parameters and features programmed while the autopilot its attached ESCs is connected to a PC. This BLHeli pass-through protocol allows you to configure and upgrade your ESCs without having to disconnect them from your vehicle. You can plug a USB cable into your autopilot and run the BLHeliSuite or BLHeliSuite32 software for Windows to configure your ESCs. 
```

Comment:
while the autopilot attached to the ESCs

Reply:
thought that was explicitly mentioned in previous sentence.....but will not hurt to  add to the end of that sentence also ..

Reply:
Sorry no I mean you have a grammatical error - "while the autopilot its attached ESCs is" does not scan

Reply:
Yes, I agree, "BLHeli capable ESCs have the ability to have their parameters and features programmed while the autopilot its attached ESCs is connected to a PC" should be something like:

- BLHeli capable ESCs have the ability to have their parameters and features programmed while remaining attached to the autopilot

OR

- BLHeli capable ESCs have the ability to have their parameters and features programmed while the autopilot they are connected to is connected to a PC

Reply:
fixed